packages/graph/Makefile.toml

```diff
@@ -0,0 +1,46 @@
+extend = { path = "../../.github/scripts/rust/Makefile.toml" }
+
+[tasks.crate-volume]
+private = true
+command = "docker"
+args = ["volume", "create", "--name", "hash-graph-data"]
```

Comment:
Are we using external volumes so the DB persists between `down`/`up` commands?

Reply:
Yes. You create a docker volume, which then can be reused for deployments